runtime/cere/src/lib.rs

```diff
@@ -125,7 +125,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// and set impl_version to 0. If only runtime
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
-	spec_version: 30800,
+	spec_version: 30900,
```

Comment:
Should devnet only runtime update be reflected in mainnet runtime code?

Reply:
Discussed.